### PR TITLE
godoc: add options to use index

### PR DESCRIPTION
**Pull request by agnivade**
----

~- [ ] The page (if new), does not already exist in the repo.~

~- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.~

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/godoc.md [hunk 1]
```diff
@@ -13,3 +13,11 @@
 - Serve documentation as a web server on port "6060":
 
 `godoc -http=:{{6060}}`
+
+- Create an index file:
+
+`godoc -write_index -index_files={{path/to/file}}`
+
+- Use the index file to enable search through the documentation:
```
Comment (agnivade) [inline pages/common/godoc.md]: I don't like this sentence structure. What I am trying to say is using the index file will allow you to enable the search functionality, which can be used to search through the docs.

How do I put this in a short sentence ? 

@sbrl , @pxgamer

**Comment by agnivade** [comment]
ping @pxgamer @sbrl

File: pages/common/godoc.md (hunk 1 above)
Comment (pxgamer) [reply pages/common/godoc.md]: I'm not really sure how to word it. :thinking:

Comment (agnivade) [reply pages/common/godoc.md]: That is not exactly what it does. Using the `-index` option enables the search. Without it, you wouldn't be able to search at all.

Comment (sbrl) [reply pages/common/godoc.md]: Ah, I see. What about `Use the specified index to search the docs`? That implies that you need to specify an index to use when searching, I think.

Comment (agnivade) [reply pages/common/godoc.md]: Hmm .. ok. Not the best, but still better than mine. I will take it.

**Pull request merged**